src/audio/rtnr/rtnr_stub.c

```diff
@@ -6,25 +6,56 @@
 //
 
 #include <sof/audio/rtnr/rtklib/include/RTK_MA_API.h>
+#include <sof/audio/audio_stream.h>
 #include <rtos/alloc.h>
 
 void RTKMA_API_S16_Default(void *Context, struct audio_stream_rtnr **sources,
 						struct audio_stream_rtnr *sink, int frames,
 						_Bool ref_active, int in_idx, int ref_idx,
 						int ref_32bits, int ref_shift)
-{}
+{
+	struct audio_stream sof_source;
+	struct audio_stream sof_sink;
+
+	rtnr_copy_to_sof_stream(&sof_source, sources[0]);
+	rtnr_copy_to_sof_stream(&sof_sink, sink);
+	audio_stream_copy(&sof_source, 0, &sof_sink, 0,
+			  frames * audio_stream_get_channels(&sof_sink));
+	rtnr_copy_from_sof_stream(sources[0], &sof_source);
+	rtnr_copy_from_sof_stream(sink, &sof_sink);
```

Comment:
since these repeat throughout the code, they're probably correct and it's just my lack of understanding of how this works

Reply:
Yes, I'm afraid so :)

Reply:
I've left this PR draft because I found this conversion has started to fail in both reat DUT and testbench. The audio_stream_get_channels(&sof_sink) gives 384 channels in tesbench... Need to fix something in the conversion.